<launch>
    <node pkg="path_planning" exec="trajectory_planner" name="Trajectory_planner" output="screen">
        <param name="odom_topic" value="/odom"/>
        <param name="map_topic" value="/map"/>
        <param name="scan_topic" value="/scan"/>
        <param name="initial_pose_topic" value="/initialpose"/>
    </node>
</launch>
